backend/main/src/Shared/Adapters/DB/NishikiDBTypes.ts

```diff
@@ -43,3 +43,58 @@ export type InvitationLink = {
 	linkExpiryTime: Date;
 	invitationLinkHash: string;
 };
+
+/**
+ * This is a food object structure recorded in the DB.
+ */
+export type FoodItem = {
+	FoodId: string;
+	Name: string | null;
```

Comment:
I think the type of  `Name` should be string because the name is a required prop. Or is there any case it can be null if we use dynamoDB?

Reply:
Thank you. The reason why I wrote it is it so is written in the document.
https://genesis-tech-tribe.github.io/nishiki-documents/project-document/database#foods

You are right, the name of the food is mandatory. I'll fix it and the document also.

Reply:
OMG, I'm so sorry. I overlooked it when I saw the document PR.